ush/fv3gfs_driver_grid.sh

```diff
@@ -479,11 +515,31 @@ fi
 #------------------------------------------------------------------------------------
 #------------------------------------------------------------------------------------
 
-export WORK_DIR=$TEMP_DIR/sfcfields
-export SAVE_DIR=$out_dir/fix_sfc
-export BASE_DIR=$home_dir
-export FIX_FV3=$out_dir
-export input_sfc_climo_dir=$home_dir/fix/sfc_climo
+
+ 
+
+	 if [[ ! -z "$ocn" ]]; then
+
+		export SAVE_DIR=$(dirname $out_dir)/C${res}.mx${ocn}/fix_sfc
+		export FIX_FV3=$(dirname $out_dir)/C${res}.mx${ocn}
+		export mosaic_file=$FIX_FV3/C${res}_mosaic.nc
+
+
+	 else
+	
+		export SAVE_DIR=$out_dir/fix_sfc
+		export FIX_FV3=$out_dir
+
+ 	 fi
+	 export WORK_DIR=$TEMP_DIR/sfcfields
+	 export BASE_DIR=$home_dir
+	 export input_sfc_climo_dir=$home_dir/fix/sfc_climo
+
+source ${BASE_DIR}/sorc/machine-setup.sh > /dev/null 2>&1
+module use ${BASE_DIR}/modulefiles
+module load build.hera.intel
+module list
+
```

Comment:
Do these modules need to be loaded again in this script?

Reply:
Hi George, I tried commenting these and I got this error ""error while loading shared libraries: libiomp5.so: cannot open shared object file:"
I will remove the module list for the time being